Unix/tests/util/util.c

```diff
@@ -401,15 +401,24 @@ MI_Result StartOmiserver()
     //
     {
         char buf[64];
+
+        int pid = -1;
+        int status = PIDFile_Read(&pid);
+
+fprintf(stderr, "pid is %d\n", pid);
```

Comment:
An error message in the else case would be more useful when the pid can't be read.